openhands/integrations/bitbucket/bitbucket_service.py

```diff
@@ -551,6 +552,83 @@ async def get_branches(self, repository: str) -> list[Branch]:
 
         return branches
 
+    async def get_paginated_branches(
```

Comment:
Would it be a good idea for these get_paginated_branches functions to have direct unit tests?